addons/account/models/account_move.py

```diff
@@ -1354,7 +1356,7 @@ def _check_tax_lock_date(self):
             JOIN res_company company ON company.id = journal.company_id
             WHERE move.id IN %s
             AND move.date < company.tax_lock_date
-        ''', [tuple(self.ids)])
+        ''', [tuple(moves.ids)])
```

Comment:
Is the term clear?  Because this would mean that someone could still add something the the sales revenue without taxes while the sales revenue needs to be reported e.g.  You seem to give a lot more freedom here.